apps/oracle-node-app/src/main/java/bisq/oracle_node/bisq1_bridge/Bisq1BridgeStore.java

```diff
@@ -37,16 +39,14 @@
  * persisted as encrypted entries and the decryption key is help by another bonded role. So it would require the
  * cooperation of the oracle node operator with the key holder.
  */
+@NoArgsConstructor(access = AccessLevel.PACKAGE)
```

Comment:
I personally don't like that much the `NoArgsConstructor` as for instance you cannot navigate to the caller with `find usage`. But thats for sure personal opinions... just wanted to mention. 
In general we tend to not over-use the lombok features and stick to the more common ones.

No need to change that, just wanted to share my opinion.

Reply:
Noted. Thanks. I appreciate learning the project's preferences.